INSTALL.md

```diff
@@ -7,6 +7,8 @@ This file contains information on how to configure and run the BPDM applications
 BPDM Pool is a SpringBoot Kotlin software project managed by Maven.
 
 The project can be run with the following command: `mvn clean spring-boot:run`
```

Comment:
Instructin should be similar to:

root: `mvn clean install`

Go to the module subfolder you like to run and use the following command: `mvn spring-boot:run`